css/src/components/notification.scss

```diff
@@ -37,71 +40,70 @@ $notification-dismiss-margin: $spacer-2 !default;
 		&.notification-#{$name} {
 			border-color: $dark;
 			background-color: $background;
-			color: $dark;
 
 			&.is-loading::before {
 				border-color: transparent transparent $dark $dark;
 			}
+
+			.notification-icon {
+				color: $dark;
```

Comment:
Fluent's figma shows the icon size as 16px. Our default icon size is 1em, so 14px here. Do we need to make an adjustment?

Reply:
Yes, I will adjust the size. Great catch ;)

Reply:
Updated